src/asset-descriptions/asset-descriptions.service.ts

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Injectable } from '@nestjs/common';
+import {
+  Asset,
+  AssetDescription,
+  AssetDescriptionType,
+  Transaction,
+} from '@prisma/client';
+import { BasePrismaClient } from '../prisma/types/base-prisma-client';
+
+@Injectable()
+export class AssetDescriptionsService {
+  async create(
+    type: AssetDescriptionType,
+    value: bigint,
+    asset: Asset,
+    transaction: Transaction,
+    prisma: BasePrismaClient,
```

Comment:
Why use the `BasePrismaClient` here and not the `PrismaService`?

Reply:
I'm going to add a loader in a subsequent PR which will pass this argument from a database transaction. We won't need that service just yet, but will later. I'll add it when we need it.